### DISCUSSION TITLE
How many developers use Homebrew in total?

### DISCUSSION THREAD

**Discussion by unrealapex** [Homebrew/discussions#4266] [Everyday usage] [answered]
### Output of `brew config`

```shell
N/A
```

### Output of `brew doctor`

```shell
N/A
```

### Description of issue

This is not a question related to any problem with Homebrew, but I was wondering how many people use Homebrew(and Linuxbrew) in total? Judging by Homebrew's analytics, I can guess somewhere in the millions.

**Comment by iMichka** [comment, marked as answer]
It’s a tricky question. Some users have disabled analytics, so we don’t have the exact numbers.

Our analytics is mainly aimed at counting package installs, not users.
